frontend/src/components/MapView/LeftPanel/AnalysisPanel/index.tsx

```diff
@@ -457,6 +522,10 @@ function AnalysisPanel({
     }
   };
 
+  if (tabIndex !== tabValue) {
```

Comment:
isn't this duplicative of what's happening above?

Reply:
No. We want this component's state "alive" while we keep jumping pages, in order to have memory of what we ware viewing there. This makes sure that nothing will render while we are not viewing this tab